staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_test.go

```diff
@@ -127,7 +127,7 @@ func newTestWatchCache(capacity int, indexers *cache.Indexers) *testWatchCache {
 	wc.stopCh = make(chan struct{})
 	pr := newConditionalProgressRequester(wc.RequestWatchProgress, &immediateTickerFactory{}, nil)
 	go pr.Run(wc.stopCh)
-	wc.watchCache = newWatchCache(keyFunc, mockHandler, getAttrsFunc, versioner, indexers, testingclock.NewFakeClock(time.Now()), schema.GroupResource{Resource: "pods"}, pr)
+	wc.watchCache = newWatchCache(keyFunc, mockHandler, getAttrsFunc, versioner, indexers, testingclock.NewFakeClock(time.Now()), defaultEventFreshDuration, schema.GroupResource{Resource: "pods"}, pr)
```

Comment:
You can set a local variable for `eventFreshDuration` equal to `defaultEventFreshDuration`. It would make it easier to test custom eventFreshDuration.

Reply:
Done.